extensions/interactions/FractionInput/FractionInput.html

```diff
@@ -0,0 +1,25 @@
+<script src="{{cache_slug}}/extensions/interactions/FractionInput/FractionInput.js"></script>
+<script src="{{cache_slug}}/extensions/interactions/FractionInput/FractionInputValidationService.js"></script>
+
+<script type="text/ng-template" id="interaction/FractionInput">
+    <form ng-submit="submitAnswer(answer)" role="form" class="form-horizontal">
+        <!-- This ng-form is needed for submitting answer on hitting enter. -->
+        <ng-form name="FractionInputForm">
+          <schema-based-editor schema="FRACTION_INPUT_FORM_SCHEMA" local-value="answer" label-for-focus-target="labelForFocusTarget">
+          </schema-based-editor>
+        </ng-form>
+
+        <div class="oppia-form-error">
```

Comment:
Could you add a bit of space between this and the answer field, so that it looks nice? It's currently a bit close.

Reply:
Hi, @seanlip is the way I've completed this one okay? I wasn't sure whether to add a new class in conjunction with the form error class or just replace the form error class entirely.

Reply:
I think it's fine, it's also OK to do an inline style here.

Reply:
(Also, the form error class seems to be used in other places, so if we were to modify that we would need to be careful to ensure that those places are behaving correctly after the change.)